pull-request.sh

```diff
@@ -254,7 +262,20 @@ main () {
                                 "${ASSIGNEES}" "${REVIEWERS}" "${TEAM_REVIEWERS}"
 
         fi
+    fi
+}
 
+# Run curl with default values
+safe_curl() {
+    printf "curl -fsSL -H 'AUTH...' %s\n" "$*" >&2
```

Comment:
What makes this safe?

Reply:
At this point it's just a curl wrapper. Maybe `do_curl`?

Reply:
How about just `curl_wrapper` if that's what the function is doing?